code/script.js

```diff
@@ -17,6 +18,8 @@ const showMessage = (message, sender) => {
       </section>
     `
   } else if (sender === 'bot') {
+    console.log(message);
```

Comment:
console.log can be removed once done with project

Reply:
Thx! :)